routes/originConda.js

```diff
@@ -35,66 +34,75 @@ async function fetchCondaRepoData(channel, subdir) {
   return repoData
 }
 
-router.get(
-  '/:channel/:subdir/:name/revisions',
-  asyncMiddleware(async (request, response) => {
-    let { channel, subdir, name } = request.params
-    channel = encodeURIComponent(channel)
-    subdir = encodeURIComponent(subdir)
-    name = encodeURIComponent(channel)
-    if (!condaChannels[channel]) {
-      return response.status(404).send(`Unrecognized Conda channel ${channel}`)
-    }
-    let channelData = await fetchCondaChannelData(channel)
-    if (!channelData.packages[name]) {
-      return response.status(404).send(`Package ${name} not found in Conda channel ${channel}`)
-    }
-    if (subdir !== '-' && !channelData.subdirs.find(x => x == subdir)) {
-      return response.status(404).send(`Subdir ${subdir} is non-existent in Conda channel ${channel}`)
-    }
-    let revisions = []
-    let subdirs = subdir === '-' ? channelData.packages[name].subdirs : [subdir]
-    for (let subdir of subdirs) {
-      const repoData = await fetchCondaRepoData(channel, subdir)
-      if (repoData['packages']) {
-        Object.entries(repoData['packages']).forEach(([, packageData]) => {
-          if (packageData.name === name) {
-            revisions.push(`${subdir}:${packageData.version}-${packageData.build}`)
-          }
-        })
-      }
-      if (repoData['packages.conda']) {
-        Object.entries(repoData['packages.conda']).forEach(([, packageData]) => {
-          if (packageData.name === name) {
-            revisions.push(`${subdir}:${packageData.version}-${packageData.build}`)
-          }
-        })
-      }
-    }
-    return response.status(200).send(uniq(revisions))
-  })
-)
+router.get('/:channel/:subdir/:name/revisions', asyncMiddleware(getOriginCondaRevisions()))
```

Comment:
take out the (), function not function invocation: `asyncMiddleware(getOriginCondaRevisions)`

Reply:
Fixed by https://github.com/clearlydefined/service/pull/1113